### PR TITLE
1370 libxml thread safe

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

**Comment by rosiereflo** [comment]
@gautierbureau do you have an idea why libxml2 library is not found in the compilation? It works fine in my local environment

File: dynawo/sources/API/CRT/test/CMakeLists.txt [hunk 1]
```diff
@@ -21,6 +22,16 @@ target_link_libraries(${MODULE_NAME}
...
 
+if(CXX11_ENABLED)
+target_link_libraries(${MODULE_NAME}
+     xml2
+  )
+  target_include_directories(${MODULE_NAME} SYSTEM
+    PRIVATE
+      $<TARGET_PROPERTY:LibXml2::LibXml2,INTERFACE_INCLUDE_DIRECTORIES>
+  )
+endif()
```
Comment (lecourtoisflo) [inline dynawo/sources/API/CRT/test/CMakeLists.txt:33]: Alineas not compliant with each other

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

File: dynawo/CMakeLists.txt [hunk 2]
```diff
@@ -272,6 +272,7 @@ if(BUILD_TESTS OR BUILD_TESTS_COVERAGE)
   add_library(dynawo_Test INTERFACE)
   target_link_libraries(dynawo_Test INTERFACE
     dynawo_Common
+    dynawo_Test_utils
```
Comment (lecourtoisflo) [inline dynawo/CMakeLists.txt:275]: isn't necessary only in case of BUILD_TESTS or BUILD_TESTS_COVERAGE ?

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

File: dynawo/CMakeLists.txt (hunk 2 above)
Comment (rosiereflo) [reply dynawo/CMakeLists.txt:275]: you are right, it is protected above (line 253)

File: dynawo/sources/Test/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
```
Comment (gautierbureau) [inline dynawo/sources/Test/CMakeLists.txt:1]: 2021

**Review by gautierbureau — APPROVED** [review]

File: dynawo/sources/Test/CMakeLists.txt (hunk 3 above)
Comment (rosiereflo) [reply dynawo/sources/Test/CMakeLists.txt]: fixed

**Review by lecourtoisflo — APPROVED** [review]

**Pull request merged**